salt/engines/slack.py

```diff
@@ -683,11 +683,11 @@ def run_commands_from_slack_async(self, message_generator, fire_all, tag, contro
                 # 10 times without taking a break.
                 log.trace('Got a message from the generator: %s', msg.keys())
                 if count > 10:
-                    log.warn('Breaking in getting messages because count is exceeded')
+                    log.warning('Breaking in getting messages because count is exceeded')
                     break
                 if len(msg) == 0:
                     count += 1
-                    log.warn('len(msg) is zero')
+                    log.warning('len(msg) is zero')
```

Comment:
Please rephrase this. Without knowing the source code this message is just useless. What does it says?